src/conf/startup.properties

```diff
@@ -168,9 +168,20 @@ prism.configstore.listeners=org.apache.falcon.entity.v0.EntityGraph,\
 *.falcon.graph.blueprints.graph=com.thinkaurelius.titan.core.TitanFactory
 
 # Graph Storage
-*.falcon.graph.storage.directory=/${falcon.home}/data/graphdb
-*.falcon.graph.storage.backend=berkeleyje
+# Berkeley DB
+#*.falcon.graph.storage.directory=/${falcon.home}/data/graphdb
+#*.falcon.graph.storage.backend=berkeleyje
+
+
+# Enabled the following for HBase
+*.falcon.graph.storage.backend=hbase
```

Comment:
We should not have either of them enabled by default in the release versions and have users download the necessary jars.  Have updated the docs